src/endpoint/sts/ops/sts_post_assume_role.js

```diff
@@ -12,6 +12,9 @@ const { CONTENT_TYPE_APP_FORM_URLENCODED } = require('../../../util/http_utils')
  */
 async function assume_role(req) {
     dbg.log1('sts_post_assume_role body: ', req.body);
+    const duration_ms = _parse_sts_duration(req.body.duration_seconds);
+    const duration_sec = Math.ceil(duration_ms / 1000);
+    const expiration_time_sec = Math.ceil((Date.now() + duration_ms) / 1000);
```

Comment:
see previous comment for reason
```suggestion
    const expiration_time = Date.now() + duration_ms;
```